src/nvim/ui.c

```diff
@@ -189,12 +189,12 @@ void ui_refresh(void)
     UI *ui = uis[i];
     width = MIN(ui->width, width);
     height = MIN(ui->height, height);
-    for (UIExtension i = 0; (int)i < kUIExtCount; i++) {
-      ext_widgets[i] &= ui->ui_ext[i];
+    for (UIExtension j = 0; (int)j < kUIExtCount; j++) {
+      ext_widgets[j] &= ui->ui_ext[j];
```

Comment:
Potential bug: if multiple UIs were attached, only the first UI would be checked to decide if a widget is externalized.

Reply:
I don't think so, a shadow binding should leave the shadowed variable unchanged. (It should still be changed of course)